Sources/VLCMediaViewEditCell.swift

```diff
@@ -0,0 +1,131 @@
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+struct VLCCheckView {
+    var isEnabled: Bool {
+        didSet {
+            let backgroundColor: UIColor = isEnabled ? .orange : .clear
+            let borderColor: UIColor = isEnabled ? .clear : .lightGray
+            view.backgroundColor = backgroundColor
+            view.layer.borderColor = borderColor.cgColor
+        }
+    }
+    var view: UIView
+}
+
+class VLCMediaViewEditCell: UICollectionViewCell {
+
+    static let identifier = String(describing: VLCMediaViewEditCell.self)
+
+    static let height: CGFloat = 56
+
+    var checkView: VLCCheckView = {
+        var view = UIView(frame: CGRect(x: 0, y: 0, width: 20, height: 20))
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.clipsToBounds = true
+        view.layer.cornerRadius = view.frame.width / 2
+        view.layer.borderColor = UIColor.lightGray.cgColor
+        view.layer.borderWidth = 1
+        return VLCCheckView(isEnabled: false, view: view)
+    }()
+
+    let thumbnail: UIImageView = {
+        let thumbnail = UIImageView()
+        thumbnail.translatesAutoresizingMaskIntoConstraints = false
+        thumbnail.contentMode = .scaleAspectFit
+        thumbnail.clipsToBounds = true
+        thumbnail.layer.cornerRadius = 3
+        return thumbnail
+    }()
+
+    let title: UILabel = {
+        let title = UILabel()
+        title.textColor = PresentationTheme.current.colors.cellTextColor
+        title.font = UIFont.systemFont(ofSize: 17)
+        title.translatesAutoresizingMaskIntoConstraints = false
+        return title
+    }()
+
+    let subInfo: UILabel = {
+        let subInfo = UILabel()
+        subInfo.textColor = PresentationTheme.current.colors.cellTextColor
+        subInfo.font = UIFont.systemFont(ofSize: 13)
+        subInfo.translatesAutoresizingMaskIntoConstraints = false
+        return subInfo
+    }()
+
+    let size: UILabel = {
+        let size = UILabel()
+        size.textColor = PresentationTheme.current.colors.cellTextColor
+        size.font = UIFont.systemFont(ofSize: 11)
+        size.translatesAutoresizingMaskIntoConstraints = false
+        return size
+    }()
+
+    let mainStackView: UIStackView = {
+        let mainStackView = UIStackView()
+        mainStackView.spacing = 20.0
+        mainStackView.axis = .horizontal
+        mainStackView.alignment = .center
+        mainStackView.translatesAutoresizingMaskIntoConstraints = false
+        return mainStackView
+    }()
+
+    let mediaInfoStackView: UIStackView = {
+        let mediaInfoStackView = UIStackView()
+        mediaInfoStackView.spacing = 5.0
+        mediaInfoStackView.axis = .vertical
+        mediaInfoStackView.alignment = .leading
+        mediaInfoStackView.translatesAutoresizingMaskIntoConstraints = false
+        return mediaInfoStackView
+    }()
```

Comment:
Do all of these really need to be properties? I can imagine that the stackviews could just be declared locally